plugins/postcss-progressive-custom-properties/src/custom/double-position-gradients.ts

```diff
@@ -0,0 +1,56 @@
+import type { Node } from 'postcss-value-parser';
+
+export function doublePositionGradients(node: Node): Array<string> {
+	const supportConditions: Array<string> = [];
+
+	// custom matchers :
+	if (node.type === 'function' && (
+		node.value === 'conic-gradient' ||
+		node.value === 'linear-gradient' ||
+		node.value === 'radial-gradient' ||
+		node.value === 'repeating-conic-gradient' ||
+		node.value === 'repeating-linear-gradient' ||
+		node.value === 'repeating-radial-gradient'
+	)) {
```

Comment:
started an issue on this as it indeed needs more thought and a more careful design : https://github.com/csstools/postcss-plugins/issues/257

Maybe the result of this will be that we keep the existing things as-is but want to open it up and see if we come up with something better.